### PR TITLE
Notification page bug with retrieving and posting new notification

### PR DESCRIPTION
**Pull request by mdrehan369**
Fixed the notification bug due to which the page was having errors and fixed the create notification problem with the image upload problem as well. Lastly, updated the UI for the Notification details section as instructed by Arpit sir

### CODE REVIEW

File: src/components/Notification/addNotify.tsx [hunk 1]
```diff
@@ -127,71 +126,62 @@ const sizeInMb = maxImageSize/(1024*1024);
...
+      let response: any = null;
 
-       const response =await apiHelperService.createNotification(
-        {...data,"background_img":Location}
-      );
-      if(response?.data?.data)
-      {
+      if (selectedPicture) {
+        formData.append("background_img", selectedPicture);
+        const postResponse = await axiosInstance.post(
```
Comment (Divesh-singh) [inline src/components/Notification/addNotify.tsx:146]: Use api service instead of direct api call here

**Comment by Divesh-singh** [comment]
Update PR title to reflect what is the issue that is being fixed here (briefly).
Also, we have set a PR format, please follow the format for pr title (check merged PR title for reference)

**Comment by mdrehan369** [comment]
> Update PR title to reflect what is the issue that is being fixed here (briefly). Also, we have set a PR format, please follow the format for pr title (check merged PR title for reference)

how do i get my ticket number??

**Comment by Akhil-chaudhary** [comment]
There's no need to include the ticket number; the system will be up and running later.

**Comment by Akhil-chaudhary** [comment]
Also a single peer review is fine

**Pull request merged**